Assets/FancyScrollView/Examples/Sources/06_LoopTabBar/Example06.cs

```diff
@@ -0,0 +1,44 @@
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace FancyScrollView.Example06
+{
+    public class Example06 : MonoBehaviour
+    {
+        [SerializeField] ScrollView scrollView = default;
+        [SerializeField] Text selectedItemInfo = default;
+        [SerializeField] private Window[] windows = default;
+
+        private Window _currentWindow;
+        
+        void Start()
+        {
+            scrollView.OnSelectionChanged(OnSelectionChanged);
+
+            var items = Enumerable.Range(0, 3)
+                .Select(i => new ItemData($"Tab {i}"))
+                .ToList();
```

Comment:
@sgamerw
[MUST]
```cs
var items = Enumerable.Range(0, windows.Length)
    .Select(i => new ItemData($"Tab {i}"))
    .ToList();
```
or
```cs
items = windows
    .Select((_, i) => new ItemData($"Tab {i}"))
    .ToList();
```

Reply:
Thanks, I will push another request one hour later when I go to work 🙂